storybook/stories/API/props/ChartProps.ts

```diff
@@ -45,6 +45,16 @@ export const margin = {
   },
 };
 
+export const dataKey = {
+  description: 'The key or getter of a gbroup of data which should be unique in a FunnelChart',
```

Comment:
Actually, I'm still confused. This is in the common file, but is now specific to `FunnelChart` - it also isn't imported and used in the storybook above. Maybe we can just remove the specification of a chart type and call it common?

Still have typo `gbroup`

Reply:
> Actually, I'm still confused. This is in the common file, but is now specific to `FunnelChart` - it also isn't imported and used in the storybook above. Maybe we can just remove the specification of a chart type and call it common?
> 
> Still have typo `gbroup`

@ckifer Yeah, that makes sense. I changed the description and fixed the typo.